packages/system-api/src/application.ts

```diff
@@ -29,66 +29,75 @@ export interface ReadonlyApplicationProperties extends ReadonlyBaseApplicationPr
     readonly action: string;
     readonly applicationName: string;
     readonly applicationVersion: string;
+    readonly resourcePath: string;
 }
 
 
-export function loadApplicationProperties(applicationId: string) {
+function getOtherApplicationProperties(message: BaseApplicationPropertiesMessage):
```

Comment:
```suggestion
function getBaseApplicationProperties(message: BaseApplicationPropertiesMessage):
```

Reply:
Also you can define `ReadonlyBaseApplicationProperties` as:

```ts
type ReadonlyBaseApplicationProperties = Readonly<
    Pick<BaseApplicationProperties, "applicationFullId" | "pathname">
>;
```

to avoid duplicate definitions of `applicationFullId` and `pathname`.

Reply:
Regarding `Pick`: I don't see any reasons to make things less explicit. 
Regarding `getOtherApplicationProperties` – I'd better make it as it was before. Doesn't make real sense to have it in a separate function.